.github/CODEOWNERS

```
@@ -1,3 +1,5 @@
 # Github codeowners file to improve our lives when selecting users for PR's
 
+requirements/ @bradchiappetta
```

```suggestion
/requirements/ @bradchiappetta
```

Would be the root `requirements` dir. Both will work however.